src/app/services/websocket.service.ts

```diff
@@ -1,14 +1,14 @@
-import {Injectable, OnDestroy} from '@angular/core';
+import {Inject, Injectable, OnDestroy} from '@angular/core';
 import {Observable} from 'rxjs/internal/Observable';
 import {Client, Message, over, StompSubscription} from '@stomp/stompjs';
 import * as SockJS from 'sockjs-client';
-import {environment} from '../../environments/environment';
 import {filter, first, switchMap} from 'rxjs/operators';
 import {BehaviorSubject} from 'rxjs/internal/BehaviorSubject';
 import {WsConsumer} from './ws-consumer';
+import { AppConfig, APP_CONFIG } from 'src/app.config';
 
-export const WS_ENDPOINT = environment.wsEndpoint;
-export const RECONNECT_INTERVAL = environment.wsReconnectInterval;
+export var WS_ENDPOINT
```

Comment:
it is old variables, please remove it and use the config directly

Reply:
I will make this change

Reply:
fixed